packages/google-vertex/src/anthropic/edge/google-vertex-anthropic-provider-edge.ts

```diff
@@ -0,0 +1,42 @@
+import {
+  generateAuthToken,
+  GoogleCredentials,
+} from '../../edge/google-vertex-auth-edge';
+import {
+  createVertexAnthropic as createVertexAnthropicOriginal,
+  GoogleVertexAnthropicProvider,
+  GoogleVertexAnthropicProviderSettings as GoogleVertexAnthropicProviderSettingsOriginal,
+} from '../google-vertex-anthropic-provider';
+
+export type { GoogleVertexAnthropicProvider };
+
+export interface GoogleVertexAnthropicProviderSettings
+  extends GoogleVertexAnthropicProviderSettingsOriginal {
+  /**
+   * Optional. The Google credentials for the Google Cloud service account. If
+   * not provided, the Google Vertex provider will use environment variables to
+   * load the credentials.
+   */
+  googleCredentials?: GoogleCredentials;
+}
+
+export function createVertexAnthropic(
+  options: GoogleVertexAnthropicProviderSettings = {},
+): GoogleVertexAnthropicProvider {
+  return createVertexAnthropicOriginal({
+    ...options,
+    headers:
+      options.headers ??
+      (async () => ({
+        'anthropic-version': 'vertex-2023-10-16',
```

Comment:
is the anthropic version needed in both headers and body?

Reply:
Good find, it appears we need `anthropic_version: 'vertex-2023-10-16',` in the request body but we don't need this in the headers. Removed here and in node variant.